Mongle/Tuist/ProjectDescriptionHelpers/Project+Templates.swift

```diff
@@ -14,29 +14,48 @@ public extension Project {
         entitlements: Entitlements? = nil,
         schemes: [Scheme] = []
     ) -> Project {
-        
-        //TODO: Relase, Dev 버전에 맞춰 bundleSuffix를 변경해야함.
-        let bundleSuffix = "dev"
         let deploymentTarget = Environment.deploymentTarget
         var projectTargets: [Target] = []
+        
+        // MARK: TARGETS > Singing & Capabilities
+        let devSigningSettings: [String: SettingValue] = [
+            "PRODUCT_BUNDLE_IDENTIFIER": "com.mongle.dev",
+            "PROVISIONING_PROFILE_SPECIFIER": "Mongle.dev",
+            "CODE_SIGN_IDENTITY": "\(APIKey.codeSignIdentity)"
+        ]
+        let releaseSigningSettings: [String: SettingValue] = [
+            "PRODUCT_BUNDLE_IDENTIFIER": "com.mongle.release",
+            "PROVISIONING_PROFILE_SPECIFIER": "Mongle.release",
+            "CODE_SIGN_IDENTITY": "\(APIKey.codeSignIdentity)"
+        ]
```

Comment:
```suggestion
        
        enum ConfigurationType: String {
            case dev = "dev"
            case release = "release"
        }

        func signingSettings(for config: ConfigurationType) -> [String: SettingValue] {
            return [
                "PRODUCT_BUNDLE_IDENTIFIER": "com.mongle.\(config.rawValue)",
                "PROVISIONING_PROFILE_SPECIFIER": "Mongle.\(config.rawValue)",
                "CODE_SIGN_IDENTITY": "\(APIKey.codeSignIdentity)"
            ]
        }
```
enum Type을 만들어서 `.dev`와 `.release`를 받아 SettingValue를 반환 해주는 `signingSettings`함수를 만들면 어떨까요?

Reply:
오! 좋은거같습니다 ㅎㅎ 👍🏻